framework/src/node/endpoint.ts

```diff
@@ -32,6 +33,25 @@ interface InitArgs {
 	registeredModules: BaseModule[];
 }
 
+const proveEventsRequestSchema = {
+	$id: '/node/endpoint/proveEventsRequestSchema',
+	type: 'object',
+	required: ['height', 'queries'],
```

Comment:
fieldNumber is missing

Reply:
Endpoint req/res are not encoded(ie: standard JSON schema). Therefore, fieldNumber is not required and it shouldnt use dataType